analyzers/tests/SonarAnalyzer.UnitTest/TestCases/HashesShouldHaveUnpredictableSalt.cs

```diff
@@ -182,26 +182,26 @@ public void ByteArrayCases(byte[] passwordBytes)
             new PasswordDeriveBytes(passwordBytes, a5); // Compliant
         }
 
-        private byte[] GetSalt() => new byte[16];
+        private byte[] GetSalt() => new byte[1];
```

Comment:
I'm not sure why 1 as it's bound to compliant cases? It's SE rule and cross-procedural case. Let's return `null;` instead to demonstrate that we don't care.

If we need a number, let's use 8.